content/docs/manual/applications-and-endpoints.md

```diff
@@ -8,12 +8,20 @@ order: 3
 Applications and Endpoints
 ======
 
-An application refers to either your backend apps or your user’s backend application for an incoming project and outgoing project respectively. It can contain more than one endpoint. This enables users to register multiple endpoints under a single application and give Convoy the ability to fan-out an event to one or more endpoints.
+## Application
+
+An application refers to either your backend apps or your user’s backend application for an incoming project and outgoing project respectively. It can contain more than one endpoint. This enables users to register multiple endpoints under a single application and give Convoy the ability to fan out an event to one or more endpoints.
 
 ## Endpoint
 
 An endpoint is a specific destination that can receive webhook events. Each endpoint belongs to an application. An endpoint can be configured with `http_timeout`.  Both applications and endpoints can be managed over the API and the dashboard.
 
+### Endpoint authentication
+
+Endpoints, both new and existing, can now require an API key to receive events.
```

Comment:
@Youngestdev 

"Endpoint authentication is a scenario where the endpoint owner needs to specify an authentication mechanism separate from signatures to validate the webhook origin. New and existing endpoints can now require an API Key to authenticate against the endpoint. In the future, we hope to support more authentication mechanisms."

Reply:
soft.